sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_last_seen_v1/query.sql

```diff
@@ -57,7 +57,10 @@ combined AS (
     (user_id, service, device_id)
 )
 SELECT
-  submission_date,
+  -- Retaining `timestamp` to represent when the last event took place
+  -- and adding submission_date for correct partitioning in BigQuery.
+  @submission_date AS submission_date,
```

Comment:
This is the core of this fix I guess? [This run](https://workflow.telemetry.mozilla.org/log?dag_id=bqetl_fxa_events&task_id=fox_accounts_derived__fxa_users_services_devices_last_seen__v1&execution_date=2022-12-01T01%3A30%3A00%2B00%3A00) was trying to write to the previous and current `submission_date`'s instead of just the current one?

Reply:
correct.